### PR TITLE
feat(touch): supports multiple files

### PR DESCRIPTION
**Pull request by nfischer**
Does what it says. From the (updated) docs:

> touch([options,] file [, file ...])
> touch([options,] file_array)

This is consistent with the API for `ls()`, `cp()`, `mv()`, `rm()`, etc.

### CODE REVIEW

File: src/touch.js [hunk 1]
```diff
@@ -31,20 +32,18 @@ function _touch(opts, files) {
...
-  if (Array.isArray(files)) {
-    files.forEach(function(f) {
-      touchFile(opts, f);
-    });
-  } else if (typeof files === 'string') {
-    touchFile(opts, files);
-  } else {
+  if (!files)
+    common.error('no files given');
+  else if (Array.isArray(files))
```
Comment (ariporad) [inline src/touch.js]: Why is this here?

Comment (nfischer) [inline src/touch.js]: This is to do a no-op. For what it's worth, I think #352 will make this line obsolete, so we might want to wait until that's merged

Comment (ariporad) [inline src/touch.js]: @nfischer: Ok, #352 is merged.

Comment (nfischer) [inline src/touch.js]: I'll rebase and update this tonight or tomorrow. Thanks!

**Comment by nfischer** [comment]
@ariporad this should be fixed now

**Comment by ariporad** [comment]
LGTM! Will merge once Travis is done.

**Comment by ariporad** [comment]
LGTM!
